nix/sources.json

```diff
@@ -95,6 +95,18 @@
         "url": "https://github.com/input-output-hk/cardano-rest/archive/7e417e9f651cb8a9012ca975454d6b22452c42d0.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "cardano-rosetta": {
+        "branch": "SRE-127_add_server_service",
```

Comment:
Looks like this branch can change to the 0.2.0 tag now (your branch merge commit + README updates)?

Reply:
Switched.